phpunit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Build a standalone phpunit without symfony/yaml
+
```

Comment:
you should add ``set -e`` to make the script fail as soon as one of the instructions is failing